lib/helpers.js

```diff
@@ -30,8 +30,12 @@ module.exports = {
             .request(method, url, request, options);
     },
 
-    readLaunchesFromFile() {
-        const files = glob.sync('rplaunch-*.tmp');
+    readLaunchesFromFile(project) {
+        const pattern = 'rplaunch-*.tmp'
+        const filesPattern = project
```

Comment:
If we read launches from a file under a specific project, does it also make sense to save them under this project? (see [this](https://github.com/reportportal/client-javascript/pull/42/files#diff-27a1c86133de4310373decfdad200a91L40) line)